src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -33550,6 +33550,30 @@ class Program
                 Diagnostic(ErrorCode.WRN_NullReferenceReturn, "F<T?>()").WithLocation(7, 28));
         }
 
+        [Fact]
```

Comment:
Consider adding a `WorkItem` attribute

Reply:
This test is for a bug that was not logged. (We were not checking nullability of the method group receiver within a delegate creation expression.)

---
In reply to: [281867678](https://github.com/dotnet/roslyn/pull/35572#discussion_r281867678) [](ancestors = 281867678)